### ISSUE TITLE
Generated Output Images information needed.

### ISSUE THREAD

**Issue by Nischit290402** [xuekt98/BBDM#23]
Hi, Can I know how to get the output image of the paired translation? In my results/.../LBBDM-f4/0/Image/train_sample directory, I have three images: Condition, ground_truth and skip_sample after training. Are we supposed to use the skip sample to obtain the desired image, or is skip_sample itself the generated image?
I would really appreciate any help I can get since I am lost with this.
P.S. : I am just trying to reimplement using the celeb dataset with masks in A and ground_truth in B. The paper does say to use VQGAN to get the final image IA->B. But I couldn't locate this specifically in the codebase.
Thank You in advance.

**Comment by xuekt98** [comment]
skip_sample is the generated image. :)

**Comment by QiFengSu** [comment]
![3@M}P@~@GW TD9{D_35@436](https://github.com/xuekt98/BBDM/assets/72723732/b6c2bf65-982a-4f83-959e-3a3a9a0a84b4)
您好，非常感谢您及您的团队能够公开代码。目前我有一个问题希望您能抽空帮我解答一下。我生成的skip_sample是这个样子的，对此我没有一点头绪，不知道您是否有时间能够帮我解答一下，感激不尽！

**Comment by QiFengSu** [comment]
> skip_sample is the generated image. :)

![3@M}P@~@GW TD9{D_35@436](https://github.com/xuekt98/BBDM/assets/72723732/b6c2bf65-982a-4f83-959e-3a3a9a0a84b4)
Hello, thank you very much for you and your team for making the code publicly available. I currently have a question and hope to get your assistance. My generated skip_sample looks like this, and I have no clue about it. I'm not sure if you have the time to help me understand it. I would greatly appreciate it!

**Comment by yug125lk** [comment]
> > skip_sample is the generated image. :)
> 
> ![3@M}P@~@GW TD9{D_35@436](https://user-images.githubusercontent.com/72723732/277164805-b6c2bf65-982a-4f83-959e-3a3a9a0a84b4.png) Hello, thank you very much for you and your team for making the code publicly available. I currently have a question and hope to get your assistance. My generated skip_sample looks like this, and I have no clue about it. I'm not sure if you have the time to help me understand it. I would greatly appreciate it!

Hi, did you solve it?

**Comment by barkindagda** [comment]
Hi, 
I am having the same results for the generated image after 10 epochs. Did anyone manage to figure out what is the reason for this ?

Thanks

**Comment by collinsrakowski** [comment]
I had the same issue

**Comment by forhollow** [comment]
To obtain accurate skip samples results, it's essential to ensure that the pre-trained model weight files are matched exactly (ckpt_path).  For example, when using the ckpt_path for LBBDM-f16.yaml in VQGAN, I used the pre-trained weight file for vq-f16 provided by the author through the given link.

**Comment by barkindagda** [comment]
Thank you for your answer @forhollow . I am unable to access the pretrained weights provided by the authors because of the Baidu (It does not accept UK mobile number).I tried training the model with random initialisation and still got same results. I am using Night2day dataset. Do you have any advice ?

Thank you.
